kingfisher_scrapy/base_spiders/big_file_spider.py

```diff
@@ -29,7 +30,16 @@ def start_requests(self):
 
     resize_package = True
 
+    @classmethod
+    def from_crawler(cls, crawler, *args, **kwargs):
+        spider = super(BigFileSpider, cls).from_crawler(crawler, *args, **kwargs)
+
+        if not spider.data_type or spider.data_type not in ('release_package', 'record_package'):
+            raise IncoherentConfigurationError(f'{spider.data_type} data_type is not compatible with BigFileSpider')
```

Comment:
Provide more relevant feedback.

```suggestion
            raise IncoherentConfigurationError(
                f"data_type must be 'release_package' or 'record_package', not {spider.data_type!r}")
```